aadhaar_demo_auth/request.py

```diff
@@ -66,11 +71,8 @@ def __setup_xml_request(self):
         hmac_node.text = self.data.hmac
         self.request_xml_root = auth_node
 
-        # Log the raw request xml for recording purpose
-        # ToDo: Move it to its own dir for logging each request-response
-        data_ready_xml_request = etree.tostring(auth_node, pretty_print=True, encoding='UTF-8', xml_declaration=True)
-        with open('raw_request.xml', 'w+') as f:
-            f.write(data_ready_xml_request)
+        DataStore(self.aadhaar_number, self.__get_txn__(), self.cfg).store_raw_request(
+                  auth_node)
```

Comment:
You could also use python's built-in logging functionality for this purpose.

Reply:
I still retained the utility DataStore to have the logic for storing request-response at a single location. Used logging within it instead of writing files.